#!/bin/sh /usr/share/dpatch/dpatch-run
## 323-magic-add-sgf.dpatch by Eduardo Sabbatella <eduardo_sabbatella@yahoo.com.ar>
##
## DP: New magic for SGF Smart Game Format

@DPATCH@

diff -Naur file-4.21.orig/magic/Magdir/games file-4.21/magic/Magdir/games
--- file-4.21.orig/magic/Magdir/games	2004-03-22 20:24:48.000000000 +0000
+++ file-4.21/magic/Magdir/games	2007-08-26 07:59:12.000000000 +0000
@@ -155,3 +155,16 @@
 
 0	string	MComprHD	MAME CHD compressed hard disk image,
 >12	belong	x		version %lu
+
+# Type: SGF Smart Game Format
+# URL:  http://www.red-bean.com/sgf/
+# From: Eduardo Sabbatella <eduardo_sabbatella@yahoo.com.ar>
+
+2	regex/c	\\(;.*GM\\[[0-9]{1,2}\\]	Smart Game Format
+>2	regex/c	GM\\[1\\]			- Go Game
+>2	regex/c	GM\\[6\\]			- BackGammon Game
+>2	regex/c	GM\\[11\\]			- Hex Game
+>2	regex/c	GM\\[18\\]			- Amazons Game
+>2	regex/c	GM\\[19\\]			- Octi Game
+>2	regex/c	GM\\[20\\]			- Gess Game
+>2	regex/c	GM\\[21\\]			- twix Game
